react/components/SearchBar/styles.css

```diff
@@ -27,6 +30,18 @@
   border-radius: 1px;
 }
 
+.searchBarInnerContainer {
```

Comment:
When using `css-handles` you don't need to add the handles to the .css file!

Reply:
Awesome! Will remove empty classes!